app/page.tsx

```
@@ -127,11 +126,57 @@ const createInventoryData = (): InventoryItem[] => {
+       return;
+
+    const commentElement = client.getCommentElement();
+    const contactElement = client.getContactElement();
+
+    commentElement.enableCustomAutocompleteSearch();
+
+    contactElement.updateContactList(users);
+
+    const subscription = commentElement
+      .on("autocompleteSearch")
+      .subscribe(async (inputData: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)